docs/diagrams/SequenceDiagram.puml

```diff
@@ -0,0 +1,111 @@
+@startuml
+actor User as User
+
+
+User -> Duke : Launches App
+activate Duke
+
+Duke -> Manager : Manager()
+activate Manager
```

Comment:
Is this wrong notation for execution of constructor? The activation bar should jus under the class. You may miss "create Manager"